src/Intent/Intent.php

```diff
@@ -46,4 +46,46 @@ public static function fromAmazonRequest(array $amazonRequest): Intent
 
         return $intent;
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function jsonSerialize()
+    {
+        $data = [];
+
+        $data['name'] = $this->name;
+        if (null !== $this->confirmationStatus)
+        {
```

Comment:
Codestyle is to add the opening brace to the end of line for `if` or `for` and not in a new line. `if (...) {` can you please fix this for all files.

Reply:
Done 👍